internal/component/azure/servicebus/subscription.go

```diff
@@ -175,7 +176,11 @@ func (s *Subscription) ReceiveAndBlock(handler HandlerFunc, lockRenewalInSec int
 		s.logger.Debugf("Received message: %s; current active message usage: %d/%d", msg.MessageID, len(s.activeMessagesChan), cap(s.activeMessagesChan))
 		// s.logger.Debugf("Message body: %s", string(msg.Body))
 
-		s.addActiveMessage(msg)
+		if err = s.addActiveMessage(msg); err != nil {
+			// If we cannot add the message then it is invalid and we cannot process it.
+			s.logger.Errorf("Error adding message: %s", err.Error())
```

Comment:
```suggestion
			s.logger.Errorf("error adding message: %s", err.Error())
```

Reply:
Should msg id be part of the error message?

Reply:
Interesting, we might need to change all error returns and logs to not start with capital letters in this file...

Reply:
Conventions with Go is that error messages (everything that is inside `errors.New` or `fmt.Errorf` or similar) should start with lowercase letters. In Dapr, our log messages (such as `logger.Errorf`) normally start with an uppercase letter. Although, personally, I'm not picky about that :)

Reply:
I am neutral either way, whether we have the convention or not, or whether the convention is for lowercased or uppercased.

Reply:
Happy to change all errors to lowercase to conform with the go convention 👌